Multiple typedef redefition warnings

Configuring the build to use Clang will result in many errors like the following:

```
/usr/include/wpe-webkit-1.1/wpe/WebKitContextMenuItem.h:41:46: warning: redefinition of typedef 'WebKitContextMenuItem' is a C11 feature [-Wtypedef-redefinition]
typedef struct _WebKitContextMenuItem        WebKitContextMenuItem;
                                             ^
/usr/include/wpe-webkit-1.1/wpe/WebKitContextMenu.h:43:39: note: previous definition is here
typedef struct _WebKitContextMenuItem WebKitContextMenuItem;
                                      ^
```

These come from the system headers for WebKit and libsoup, which seem to assume that the compiler that will be used will be run with `-std=c11` (or similar) to enable C11 mode.

--- a/meson.build
+++ b/meson.build
@@ -3,7 +3,7 @@ project('cog', 'c',
     default_options: [
         'buildtype=debugoptimized',
         'b_ndebug=if-release',
-        'c_std=c99',
+        'c_std=c11',
     ],
     license: 'MIT',
     version: '0.17.0',

